code/datums/components/swabbing.dm

```diff
@@ -0,0 +1,114 @@
+/*!
+
+This component is used in vat growing to swab for microbiological samples which can then be mixed with reagents in a petridish to create a culture plate.
+
+*/
+/datum/component/swabbing
+	///The current datums on the swab
+	var/list/swabbed_items
```

Comment:
This is a list of references.

Can they be deleted before the component is deleted? If so, register on their destruction.

Clear the list once the component is destroyed.

Reply:
nooo you cant just let instances float around without being referenced anywhere

hahaha GC machine go bRRRRRRRR